### DISCUSSION TITLE
How to filter data features into GeoJsonLayer with DataFilterExtension

### DISCUSSION THREAD

**Discussion by epplestun** [visgl/deck.gl#6305] [General]
It's posible to filter GeoJsonLayer data with other criteria different to filterRange?, for example with something like filterContains?

**Comment by AdriSolid** [comment]
Do you mean filtering qualitative values? If so your `filterRange` could be `[1, 1]`, then check with `getFilterValue` if your feature contains whatever, then return `1` (true) or `0` (false), something like:

```javascript
new Layer({
  ...
  extensions: [new DataFilterExtension()],
  filterRange: [1, 1],
  getFilterValue: (f) => {
    if (f.properties.yourColumn === 'some-string') {
      return 1; // true
    }
    return 0; // false
  }
});
```

**Reply by epplestun** [reply]
Sounds good! I'll try :) thanks!
